doc/source/news.rst

```diff
@@ -7,6 +7,70 @@
 News
 ====
 
+.. _release-9-0-5:
+
+Release 9.0.5 - 2019-07-29
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`reference/commands/logical_range_filter`] Stop an optimization by the number of required records.
+
+  * This feature is used only when the target shard is large enough.
+  * Large enough threshold is 10000 by default.
+
+* [:doc:`/reference/normalizers`] Added new option ``unify_to_katakana`` for ``NormalizerNFKC100``.
+
+  * This option normalize hiragana to katakana.
+  * For example, ``ゔぁゔぃゔゔぇゔぉ`` normalize to ``ヴァヴィヴヴェヴォ``.
+
+* [:doc:`reference/commands/select`] Added support drilldowns for a slice parameter.
+
```

Comment:
`Added drilldowns support as a slice parameter` or `Added support for drilldowns as a slice parameter` ?